package.json

```diff
@@ -63,7 +64,8 @@
     "test:single": "mocha -timeout 180000 --full-trace",
     "test:system": "mocha -timeout 180000 --recursive --full-trace system_test/*.js",
     "test:unit": "mocha -timeout 180000 --recursive --full-trace test/unit/**/*.js test/unit/*.js",
-    "test:ci": "mocha -timeout 180000 --recursive --full-trace test/**/*.js system_test/*.js"
+    "test:ci": "mocha -timeout 180000 --recursive --full-trace  test/**/**/*.js test/**/*.js system_test/*.js",
```

Comment:
maybe let's split test:ci and test:coverage to include or not system_test directory since it works only on internal Snowflake builds

Reply:
now we have scripts: "test:ci:coverage": "nyc npm run test:ci" and "test:unit:coverage"